### DISCUSSION TITLE
Upscaling and downscaling based on different queries.

### DISCUSSION THREAD

**Discussion by Savitha-Amaldev** [kedacore/keda#2170] [Q&A / Need Help] [answered]
I have a scaled object which does autoscaling based on an MSSQL trigger. But according to my requirement, I want my services to be spun up based on a query, and downscaling of the same pods should happen based on a different SQL query.

How can I achieve this? Please help me here

**Comment by tomkerkhove** [comment, marked as answer]
Unfortunately that is not possible, so let's open a feature request!

**Reply by tomkerkhove** [reply]
Opened https://github.com/kedacore/keda/issues/2172 and I'm assuming you are referring to horizontal scaling and not vertical? Because scaling up/down is vertical scaling which we do not support.

**Reply by Savitha-Amaldev** [reply]
I'm referring to horizontal scaling only. Just increase the pod count based on criteria and decrease the pod count based on different criteria

**Reply by tomkerkhove** [reply]
Scaling out/in then, perfect.
